.github/workflows/release_binary.yml

```diff
@@ -0,0 +1,84 @@
+name: Binary release
+
+# Manual release with input version number
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Enter the release version (e.g., 0.1.0)
+        required: true
+  push:
+    branches:
+      - binary-release
+
+jobs:
+  # This job builds the binaries for each release
+  build-binary:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        include:
+          - os: ubuntu-latest
+            artifact_name: linux-artifact
+            artifact_filename: stump_server
+          - os: windows-latest
+            artifact_name: windows-artifact
+            artifact_filename: stump_server.exe
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Build web application
+        uses: ./.github/actions/build-web
```

Comment:
Based on the latest CI run, uploading with the same key within the context of the same workflow seems to be problematic:

```bash
Error: Failed to CreateArtifact: Received non-retryable error: Failed request: (409) Conflict: an artifact with this name already exists on the workflow run
```

Since it is just a bundle of CSS/JS/etc, it shouldn't _really_ matter if the build happens within the matrix here as it should be the same across platforms. It might be acceptable to move this step to a separate job in this workflow just for the webapp (e.g. `build-webapp`)